presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java

```diff
@@ -198,6 +200,17 @@ public void testFromUnixTimeWithOffset()
         assertFunction("from_unixtime(" + seconds + ", " + timeZoneHoursOffset + ", " + timezoneMinutesOffset + ")", TIMESTAMP_WITH_TIME_ZONE, toTimestampWithTimeZone(expected));
     }
 
+    @Test
```

Comment:
cmt msg:
replace `Categroize error for from_unixtime` with `Categorize invalid offset minutes error in from_unixtime`

Reply:
You mean the pull request title? I can't find the "Categroize error for from_unixtime" string in this test...

Reply:
I have updated the commit message to Categorize invalid offset minutes error in from_unixtime